app/src/main/java/org/oppia/app/topic/TopicTab.kt

```diff
@@ -0,0 +1,19 @@
+package org.oppia.app.topic
+
+enum class TopicTab(val position: Int) {
```

Comment:
Can ``position`` be private? Prefer minimizing visibility.

Reply:
Resolved